ca/update.sh

```diff
@@ -9,9 +9,16 @@ if test $? -gt "0"; then
   d="$(date +%Y-%m-%d)"
   cp cacert.pem "cacert-${d}.pem"
   sha256sum cacert.pem > cacert.pem.sha256
-  cp cacert.pem.sha256 "cacert-${d}.pem.sha256"
+  sha256sum "cacert-${d}.pem" > "cacert-${d}.pem.sha256"
```

Comment:
what is wrong with the old code that copies the hash file?

Reply:
The sha256 file contains the filename, and if that filename doesn't match you can't use sha256sum --check, so actually *using* the sha256 file is harder than necessary

Reply:
ah right i forgot about that. the rebuild should be a one-time thing so it may be easier for @bagder to just run it once

Reply:
I'd be happier with a solution that doesn't require regenerating the hash. One option is to replace `cp` with `sed`.

Reply:
That's a good point. I'll update my tree later today

Reply:
I've opened #269 with the above snippet after the post above.